tests/test_benchmark.py

```diff
@@ -978,12 +981,12 @@ def test_zzz_poppler(self):
           '5': 55,
         };
         if (benchmarkArgument === 0) {
-          Module.arguments = ['-?'];
-          Module.printErr = function(){};
+          Module['arguments'] = ['-?'];
+          Module['printErr'] = function(){};
         } else {
           // Add 'filename' after 'input.pdf' to write the output so it can be verified.
-          Module.arguments = ['-scale-to', '1024', 'input.pdf',  '-f', '1', '-l', '' + benchmarkArgumentToPageCount[benchmarkArgument]];
-          Module.postRun = function() {
+          Module['arguments'] = ['-scale-to', '1024', 'input.pdf',  '-f', '1', '-l', '' + benchmarkArgumentToPageCount[benchmarkArgument]];
+          Module['postRun'] = function() {
```

Comment:
Why this change?

Reply:
Because closure mangles `postRun` into something else, and it never gets run. Since we never bothered checking the output, the benchmark passed anyways.